samples/fortio/stackdriver.yaml

```diff
@@ -0,0 +1,245 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: handler
```

Comment:
cc @bianpengyuan : I'm not sure how to trigger this test. I think it only runs on release?

Reply:
It runs with nightly release: https://testgrid.k8s.io/istio-release#e2e-stackdriver

Reply:
It runs with nightly release: https://testgrid.k8s.io/istio-release#e2e-stackdriver

Reply:
Thanks. I see it runs on 1.1 only. Is this intentional? I was expecting the config with "logentry" kind to be rejected on master, but since it runs on 1.1 only, it passes.

Reply:
Sorry I forgot to port the test into master. https://github.com/istio/test-infra/pull/1235 should do the work. Thanks for catching this!

Reply:
No worries, we need to merge this or the test will fail on master, I think.